canonical/ubuntu.com
# Remove humans.txt file

## Done

- Remove humans.txt file since this is going to be a nightmare to keep up to date, and a lot of people are missing from here :cry: 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/humans.txt
- Should redirect to the homepage

## Issue / Card

Fixes #12313
